pytorch_lightning/metrics/regression/explained_variance.py

```diff
@@ -43,6 +83,8 @@ def update(self, preds: torch.Tensor, target: torch.Tensor):
             preds: Predictions from model
             target: Ground truth values
         """
+        assert preds.shape == target.shape, \
```

Comment:
lets consider using Error clases instead...
https://medium.com/@jadhavmanoj/python-what-is-raise-and-assert-statement-c3908697bc62#:~:text=Python's%20assert%20statement%20is%20a,a%20bug%20in%20your%20program.

Reply:
So instead `assert` it should be a `raise`?